scanapi/reporter.py

```diff
@@ -33,6 +33,17 @@ def write(self, results):
         logger.info("\nThe documentation was generated successfully.")
         logger.info(f"It is available at {abspath(self.output_path)}")
 
+    def write_without_generating_report(self, results):
```

Comment:
@JRubics can we have a [docstring](https://pythonhelp.wordpress.com/2011/02/14/docstrings/) to indicate what is this method for? The docstring is used to build our internal code documentation. I will add to the wiki that we need a docstring for every new method, this way new comers will know that too.

Example of docstring: https://github.com/scanapi/scanapi/blob/251640c7a7f99aff9cad2db6ce329ea2b8f83b7f/scanapi/reporter.py#L20

Reply:
done :)